ui/app/components/http-requests-bar-chart.js

```diff
@@ -3,10 +3,13 @@ import d3 from 'd3-selection';
 import d3Scale from 'd3-scale';
 import d3Axis from 'd3-axis';
 import d3TimeFormat from 'd3-time-format';
+import d3Tip from 'd3-tip';
+import d3Array from 'd3-array';
```

Comment:
It looks like this isn't used elsewhere? I'm guessing listing failed on this but it's not failing the build - I'll check on that, but for now I think we can remove this here and from `package.json`